src/Microsoft.DotNet.Tools.Repl.Csi/Program.cs

```diff
@@ -19,21 +24,128 @@ public static int Main(string[] args)
             app.FullName = "C# REPL";
             app.Description = "C# REPL for the .NET platform";
             app.HelpOption("-h|--help");
+
             var script = app.Argument("<SCRIPT>", "The .csx file to run. Defaults to interactive mode.");
+            var framework = app.Option("-f|--framework <FRAMEWORK>", "Compile a specific framework", CommandOptionType.MultipleValue);
+            var configuration = app.Option("-c|--configuration <CONFIGURATION>", "Configuration under which to build", CommandOptionType.SingleValue);
+            var preserveTemporary = app.Option("-t|--preserve-temporary", "Keep the output's temporary directory around", CommandOptionType.NoValue);
```

Comment:
@ManishJayaswal, it is an option supported by dotnet run, and allows the user to optionally save the temporary files for further debugging/investigation. Deleting the files after the environment closes is the current default.